lib/enum/arrays.rb

```diff
@@ -1,6 +1,11 @@
 class Array
 
+  def initialize(array)
+    @array = array
+  end
+
   def sum_of_numeric_elements
+    array.select{|object|}
```

Comment:
You're on the right track here. First, you need to select just the numeric elements, like this: `select{ |object| object.is_a? Numeric` Then you need to sum just those elements using reduce. Reduce is an accumulator, meaning you give it an operator like `+` as a symbol, and the reduce method will do that operation on each element, adding each one to the previous result, and then give you the final result. 

Putting that all together: `array.select{ |object| object.is_a? Numeric }.reduce(:+)`